client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/generic.tsx

```diff
@@ -96,10 +95,9 @@ export default function GenericThankYou( { purchases, emailAddress }: GenericTha
 		}
 	} );
 
-	// Fallback to the default generic support blurb if there less than two support blurbs in the footer.
-	if ( footerDetails.length < 2 ) {
-		footerDetails = footerDetails.concat( getDefaultFooterDetails( 'generic' ) );
-	}
+	// Note: We've disabled the default support blurb, because it's now
+	// redundant with the Help Center link in the header of Thank You pages.
+	// See https://github.com/Automattic/dotcom-forge/issues/8175 for related discussion.
 
```

Comment:
1. Let's just remove it. Anyway we have commits history, and I don't see some big benefits from such comment. Unless you see some benefits.
```suggestion
```

2. Let's also do clean up - remove the actual file with the item config `client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/content/get-default-footer-details.ts`

Reply:
Done, default footer details are fully removed